src/utils/images.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Util method to retrieve image path from /images folder from the relative file path,
+ * e.g. "/images/album/picture.jpg" -> "album/picture.jpg"
+ */
+export const getImagePath = (imageLink: string) => {
+  const cleanImagePath = decodeURI(imageLink).replace(/^\//, "")
+  const filePathArr = cleanImagePath.split("/")
+  const filePath = filePathArr[filePathArr.length - 1]
+  return filePath
```

Comment:
won't this return the last item in the array post split? ie, `"/images/album/picture.jpg" -> "picture.jpg"`
![Screenshot 2023-08-30 at 12 20 44 PM](https://github.com/isomerpages/isomercms-frontend/assets/44049504/7045c7a0-2aaa-4d7d-821f-852c72fe7eb6)

Reply:
Good catch, i only checked this on first level images - have made a fix in https://github.com/isomerpages/isomercms-frontend/pull/1461/commits/62d436c34d3bc02653de03ff1ae39fa2a27bf7b6